apps/condo/domains/user/components/UserSettingsContent.tsx

```diff
@@ -36,15 +44,58 @@ export const UserSettingsContent: React.FC = () => {
     const CompanyName = intl.formatMessage({ id: 'CompanyName' })
     const HasMarketingConsentTitle = intl.formatMessage({ id: 'pages.condo.profile.hasMarketingConsent' }, { company: CompanyName })
     const DisabledHasMarketingConsentTooltip = intl.formatMessage({ id: 'pages.condo.profile.hasMarketingConsent.disabled.tooltip' })
+    const DisableSPPTitle = intl.formatMessage({ id: 'pages.condo.profile.disableSPPTitle' })
+    const DisableSPPRequestSentTitle = intl.formatMessage({ id: 'pages.condo.profile.disableSPPRequestSentTitle' })
+    const SendDisableSPPRequestTitle = intl.formatMessage({ id: 'pages.condo.profile.sendDisableSPPRequestTitle' })
+    const SendDisableSPPRequestModalTitle = intl.formatMessage({ id: 'pages.condo.profile.sendDisableSPPRequestModalTitle' })
+    const SendDisableSPPRequestModalMessage = intl.formatMessage({ id: 'pages.condo.profile.sendDisableSPPRequestModalMessage' })
+    const SendDisableSPPRequestConfirmButtonText = intl.formatMessage({ id: 'pages.condo.profile.sendDisableSPPRequestConfirmButtonText' })
 
     const RuTitle = intl.formatMessage({ id: 'language.russian.withFlag' })
     const EnTitle = intl.formatMessage({ id: 'language.english-us.withFlag' })
     const EsTitle = intl.formatMessage({ id: 'language.spanish-es.withFlag' })
 
     const [showGlobalHints, setShowGlobalHints] = useState<boolean>(false)
     const [hasMarketingConsent, setMarketingConsent] = useState<boolean>(false)
+    const [disableSPPConfirmModalShown, setDisableSPPConfirmModalShown] = useState<boolean>(false)
+    const [isSendingDisableSPPRequest, setIsSendingDisableSPPRequest] = useState<boolean>(false)
+    const sppBillingId = sppConfig?.BillingIntegrationId || null
+    const { organization } = useOrganization()
 
     const { user, refetch } = useAuth()
+    const { useFlag } = useFeatureFlags()
+    const isSPPOrg = useFlag(SERVICE_PROVIDER_PROFILE)
+    const { objs: userHelpRequests, refetch: userHelpRequestRefetch, loading: userHelpRequestsLoading } = UserHelpRequest.useObjects({
+        where: {
+            organization: { id: organization?.id },
+            billingIntegration: { id: sppBillingId },
+            type: UserHelpRequestTypeType.IntegrationSetup,
+        },
+        first: 1,
+    }, { skip: !organization || !isSPPOrg || !sppBillingId })
+
+    const createUserHelpRequestAction = UserHelpRequest.useCreate({
+        type: UserHelpRequestTypeType.IntegrationSetup,
+    }, () => userHelpRequestRefetch())
```

Comment:
Why you use old client utilities?
We write a gql query in queries folder, then generate hooks and use it.
Pls change it or at least leave a TODO